### PR TITLE
V3 message fetch

### PR DESCRIPTION
**Pull request by elverkilde**
### What?

 * `fetchMultipartMessages` using the new v3 API
 * Fix bug in `verify`
 * Minor convenience function for creating/debugging tests

### Why?

 * Needed for v3 support

----

- [x] CHANGELOG updated if relevant?

### CODE REVIEW

**Comment by elverkilde** [comment]
[[0.5] Fetch v3 messages](https://trello.com/c/6ym6UKpC/1654-05-fetch-v3-messages)

File: tests/ClientTest.php [hunk 1]
```diff
@@ -1034,6 +1063,90 @@ public function testSendSimpleMessageShouldReturnAResponsePayloadIfARoomIDSender
...
+    {
+        $this->expectException(Chatkit\Exceptions\TypeMismatchException::class);
+        $this->chatkit->fetchMultipartMessages([ 'room_id' => 42]);
+
+        $this->expectException(Chatkit\Exceptions\TypeMismatchException::class);
+        $this->chatkit->fetchMultipartMessages([ 'room_id' => 'correct',
+                                                 'limit' => 'incorrect']);
+    }
+
+    public function testFetchMultipartMessagesShouldReturnAResponsePayloadIfARoomIDIsProvide()
```
Comment (callum-oakley) [inline tests/ClientTest.php]: typo: IsProvide**d**

**Review by callum-oakley — APPROVED** [review]
Tiny typo, otherwise LGTM.

File: tests/ClientTest.php (hunk 1 above)
Comment (elverkilde) [reply tests/ClientTest.php]: :+1:

**Pull request merged**